frontend/src/components/table/BookingsTable.tsx

```diff
@@ -74,6 +77,7 @@ export default function BookingsTable({
                 <StyledIconButton
                   className="pointer"
                   onClick={async () => {
+                    addAlert('Exporting the Excel...', null);
```

Comment:
"Exporting the ADM322 Excel file"

^ Graeme can confirm but I prefer this message

Reply:
Great!